src/dropsgear.ts

```diff
@@ -457,6 +457,21 @@ function juneCleaver(equipMode: BonusEquipMode): Map<Item, number> {
         valueJuneCleaverOption(juneCleaverChoiceValues[choice][bestJuneCleaverOption(choice)])
       ) / JuneCleaver.choices.length;
   }
+  // If we're ascending then the chances of hitting choices in the queue is reduced
+  if (globalOptions.ascending && estimatedTurns() <= 180 && JuneCleaver.getInterval() === 30) {
+    const availEV =
+      sum([...JuneCleaver.choicesAvailable()], (choice) =>
+        valueJuneCleaverOption(juneCleaverChoiceValues[choice][bestJuneCleaverOption(choice)])
+      ) / JuneCleaver.choicesAvailable().length;
+    const queueEV =
+      sum(
+        [...JuneCleaver.queue()],
+        (choice) =>
+          valueJuneCleaverOption(juneCleaverChoiceValues[choice][bestJuneCleaverOption(choice)]) *
+          ((1 - 2 / 3) ^ Math.max(0, estimatedTurns() - (1 + JuneCleaver.queue().indexOf(choice))))
```

Comment:
Okay this line is definitely at least somewhat wrong insofar as estimatedTurns() will often be very large

Reply:
Ah, it should be Math.floor(estimatedTurns()/30) instead of just estimatedTurns().

Do you have any suggestions for how to make this a bit more readable? I read the slap article but I'm not entirely clear on what a level of abstraction really means for something like this.